apps/system/js/sound_manager.js

```diff
@@ -351,16 +353,16 @@
       case 'unmute':
         this.setMute(false);
         break;
+      case 'audiochannelchangedasactive':
+        this.setAudioChannel(e.detail.channel);
+        this.ceAccumulator();
+        break;
       case 'mozChromeEvent':
         switch (e.detail.type) {
           case 'bluetooth-volumeset':
             this.changeVolume(e.detail.value - this.currentVolume.bt_sco,
                               'bt_sco');
             break;
-          case 'audio-channel-changed':
-            this.setAudioChannel(e.detail.channel);
-            this.ceAccumulator();
-            break;
```

Comment:
@evanxd Before the new audio channel api actually landed in gecko, we should keep this `audio-channel-changed` case in gaia and make sure it still works on master branch with current gecko.